src/chat.cpp

```diff
@@ -609,17 +609,23 @@ bool CHAT_Input( event_t *pEvent )
 			{
 				if (( g_ulChatMode == CHATMODE_PRIVATE_SEND ) && ( g_ulChatPlayer != MAXPLAYERS ))
 				{
-					ULONG oldPlayer = g_ulChatPlayer;
+					int tempPlayer = g_ulChatPlayer;
+					int direction = ( pEvent->data3 & GKM_SHIFT ) ? -1 : 1;
```

Comment:
Are these changes here related to any of the points I brought up?  At first glance, it doesn't look like this to me. I also just realize that github seems to have hid the changes to chat.cpp when I made the review.

Reply:
These changes aren't related to anything you mentioned last week, it's just something I forgot to add when I originally made the pull request.

Reply:
I see, sounds like a nice addition.